content/docs/reference-react-dom.md

```diff
@@ -36,23 +36,23 @@ React supports all popular browsers, including Internet Explorer 9 and above, al
 ReactDOM.render(element, container[, callback])
 ```
 
-Render a React element into the DOM in the supplied `container` and return a [reference](/docs/more-about-refs.html) to the component (or returns `null` for [stateless components](/docs/components-and-props.html#functional-and-class-components)).
+將 React element 渲染在提供的 `container` 的 DOM 之中並返回 component [reference](/docs/more-about-refs.html)（或是 [stateless components](/docs/components-and-props.html#functional-and-class-components) 則返回 `null`）。
 
-If the React element was previously rendered into `container`, this will perform an update on it and only mutate the DOM as necessary to reflect the latest React element.
+如果 React element 已經在 `container` 中渲染，這將對它執行更新並只在必要時改變 DOM 以反映出最新的 React element。
 
-If the optional callback is provided, it will be executed after the component is rendered or updated.
+如果提供了可選的 callback，它將會在 component 渲染或更新之後被執行。
```

Comment:
Here is the reason I translated `render` into `渲染` if I keep `render` but not translated `update` is weird because they describe the same thing.

Reply:
Hey @abrcdf1023 I understand that you think it might look weird to translate one but not the other. But the community thinks that [渲染](https://zh.wikipedia.org/wiki/%E6%B8%B2%E6%9F%93) is often confused with `rendering` in computer graphics. 

For the clarity and consistency across the entire document, let's keep it as render. :)

Reply:
@chloewlin Ok,I have fixed it please take a look.